### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by ssriram1992** [lgottwald/PaPILO#40]
I tried installing SCIPOptSuite 7.0.1 in Windows Subsystem linux - Ubuntu 18.04 using a cmake install with all default settings. I got the following error:
```
/usr/bin/ld: /usr/local/lib/libtbb.a(governor.o): undefined reference to symbol 'pthread_key_delete@@GLIBC_2.2.5'
//lib/x86_64-linux-gnu/libpthread.so.0: error adding symbols: DSO missing from command line  
collect2: error: ld returned 1 exit status  
make[2]: *** [bin/scip] Error 1      
make[1]: *** [scip/src/CMakeFiles/scip.dir/all] Error 2   
make: *** [all] Error 2    
```
Through personal emails, I was told that this is a PaPILO issue. As expected, running the cmake after disabling PaPILO and then installing scip worked.  Please let me know if there is any additional information that you would like me to provide.

**Comment by ssriram1992** [comment]
Opening the issue as requested by @fschloesser

**Comment by lgottwald** [comment]
Thanks for opening the issue. The problem has to do with the way tbb is used. Papilo tries to look for the tbb library in the system, and only when it is not found it will try to build tbb by itself as a static library. From the error message I see that tbb was built as a static library and that there must be some problem with the linking step, as the tbb library requires pthread. An workaround to make this go a way is to provide an installed tbb library.

For fixing this I need a bit more information. I would like to see the linker command which you can see when you compile with `make VERBOSE=true`. The DSO missing kind of error can come up when the order in which the libraries are linked is not correct. Please post the full command that is used for linking the binary that currently failed. This will help me to change that command by hand so that you can try if it fixes the issue, and then adapt the built system to produce the correct command in the first place.

**Comment by ssriram1992** [comment]
Sure, here is the last output before error.
```
[ 40%] Linking CXX executable ../../bin/scip
cd /opt/scipoptsuite-7.0.1/papBuild/scip/src && /usr/bin/cmake -E cmake_link_script CMakeFiles/scip.dir/link.txt --verbose=true
/usr/bin/c++   -Wno-psabi -Wno-psabi -DBLISS_USE_GMP -O3 -DNDEBUG  -rdynamic CMakeFiles/scip.dir/main.c.o CMakeFiles/scip.dir/__/__/scipbuildflag.c.o CMakeFiles/scip.dir/blockmemshell/memory.c.o CMakeFiles/scip.dir/scip/message.c.o CMakeFiles/scip.dir/scip/bitencode.c.o CMakeFiles/scip.dir/scip/bandit.c.o CMakeFiles/scip.dir/scip/bandit_epsgreedy.c.o CMakeFiles/scip.dir/scip/bandit_exp3.c.o CMakeFiles/scip.dir/scip/bandit_ucb.c.o CMakeFiles/scip.dir/scip/branch_allfullstrong.c.o CMakeFiles/scip.dir/scip/branch_cloud.c.o CMakeFiles/scip.dir/scip/branch_distribution.c.o CMakeFiles/scip.dir/scip/branch_fullstrong.c.o CMakeFiles/scip.dir/scip/branch_inference.c.o CMakeFiles/scip.dir/scip/branch_leastinf.c.o CMakeFiles/scip.dir/scip/branch_lookahead.c.o CMakeFiles/scip.dir/scip/branch_mostinf.c.o CMakeFiles/scip.dir/scip/branch_multaggr.c.o CMakeFiles/scip.dir/scip/branch_nodereopt.c.o CMakeFiles/scip.dir/scip/branch_pscost.c.o CMakeFiles/scip.dir/scip/branch_random.c.o CMakeFiles/scip.dir/scip/branch_relpscost.c.o CMakeFiles/scip.dir/scip/branch_vanillafullstrong.c.o CMakeFiles/scip.dir/scip/cons_abspower.c.o CMakeFiles/scip.dir/scip/compr_largestrepr.c.o CMakeFiles/scip.dir/scip/compr_weakcompr.c.o CMakeFiles/scip.dir/scip/concsolver_scip.c.o CMakeFiles/scip.dir/scip/cons_and.c.o CMakeFiles/scip.dir/scip/cons_bivariate.c.o CMakeFiles/scip.dir/scip/cons_benders.c.o CMakeFiles/scip.dir/scip/cons_benderslp.c.o CMakeFiles/scip.dir/scip/cons_bounddisjunction.c.o CMakeFiles/scip.dir/scip/cons_cardinality.c.o CMakeFiles/scip.dir/scip/cons_components.c.o CMakeFiles/scip.dir/scip/cons_conjunction.c.o CMakeFiles/scip.dir/scip/cons_countsols.c.o CMakeFiles/scip.dir/scip/cons_cumulative.c.o CMakeFiles/scip.dir/scip/cons_disjunction.c.o CMakeFiles/scip.dir/scip/cons_indicator.c.o CMakeFiles/scip.dir/scip/cons_integral.c.o CMakeFiles/scip.dir/scip/cons_knapsack.c.o CMakeFiles/scip.dir/scip/cons_linear.c.o CMakeFiles/scip.dir/scip/cons_linking.c.o CMakeFiles/scip.dir/scip/cons_logicor.c.o CMakeFiles/scip.dir/scip/cons_nonlinear.c.o CMakeFiles/scip.dir/scip/cons_or.c.o CMakeFiles/scip.dir/scip/cons_orbisack.c.o CMakeFiles/scip.dir/scip/cons_orbitope.c.o CMakeFiles/scip.dir/scip/cons_pseudoboolean.c.o CMakeFiles/scip.dir/scip/cons_quadratic.c.o CMakeFiles/scip.dir/scip/cons_setppc.c.o CMakeFiles/scip.dir/scip/cons_soc.c.o CMakeFiles/scip.dir/scip/cons_sos1.c.o CMakeFiles/scip.dir/scip/cons_sos2.c.o CMakeFiles/scip.dir/scip/cons_superindicator.c.o CMakeFiles/scip.dir/scip/cons_symresack.c.o CMakeFiles/scip.dir/scip/cons_varbound.c.o CMakeFiles/scip.dir/scip/cons_xor.c.o CMakeFiles/scip.dir/scip/dcmp.c.o CMakeFiles/scip.dir/scip/dialog_default.c.o CMakeFiles/scip.dir/scip/event_globalbnd.c.o CMakeFiles/scip.dir/scip/event_estim.c.o CMakeFiles/scip.dir/scip/event_softtimelimit.c.o CMakeFiles/scip.dir/scip/event_solvingphase.c.o CMakeFiles/scip.dir/scip/disp_default.c.o CMakeFiles/scip.dir/scip/heur_actconsdiving.c.o CMakeFiles/scip.dir/scip/heur_adaptivediving.c.o CMakeFiles/scip.dir/scip/heur_bound.c.o CMakeFiles/scip.dir/scip/heur_clique.c.o CMakeFiles/scip.dir/scip/heur_coefdiving.c.o CMakeFiles/scip.dir/scip/heur_completesol.c.o CMakeFiles/scip.dir/scip/heur_conflictdiving.c.o CMakeFiles/scip.dir/scip/heur_crossover.c.o CMakeFiles/scip.dir/scip/heur_dins.c.o CMakeFiles/scip.dir/scip/heur_distributiondiving.c.o CMakeFiles/scip.dir/scip/heur_dualval.c.o CMakeFiles/scip.dir/scip/heur_farkasdiving.c.o CMakeFiles/scip.dir/scip/heur_feaspump.c.o CMakeFiles/scip.dir/scip/heur_fixandinfer.c.o CMakeFiles/scip.dir/scip/heur_fracdiving.c.o CMakeFiles/scip.dir/scip/heur_gins.c.o CMakeFiles/scip.dir/scip/heur_guideddiving.c.o CMakeFiles/scip.dir/scip/heur_indicator.c.o CMakeFiles/scip.dir/scip/heur_intdiving.c.o CMakeFiles/scip.dir/scip/heur_intshifting.c.o CMakeFiles/scip.dir/scip/heur_linesearchdiving.c.o CMakeFiles/scip.dir/scip/heur_localbranching.c.o CMakeFiles/scip.dir/scip/heur_lpface.c.o CMakeFiles/scip.dir/scip/heur_locks.c.o CMakeFiles/scip.dir/scip/heur_alns.c.o CMakeFiles/scip.dir/scip/heur_mutation.c.o CMakeFiles/scip.dir/scip/heur_multistart.c.o CMakeFiles/scip.dir/scip/heur_mpec.c.o CMakeFiles/scip.dir/scip/heur_nlpdiving.c.o CMakeFiles/scip.dir/scip/heur_objpscostdiving.c.o CMakeFiles/scip.dir/scip/heur_octane.c.o CMakeFiles/scip.dir/scip/heur_ofins.c.o CMakeFiles/scip.dir/scip/heur_oneopt.c.o CMakeFiles/scip.dir/scip/heur_padm.c.o CMakeFiles/scip.dir/scip/heur_proximity.c.o CMakeFiles/scip.dir/scip/heur_pscostdiving.c.o CMakeFiles/scip.dir/scip/heur_reoptsols.c.o CMakeFiles/scip.dir/scip/heur_repair.c.o CMakeFiles/scip.dir/scip/heur_randrounding.c.o CMakeFiles/scip.dir/scip/heur_rens.c.o CMakeFiles/scip.dir/scip/heur_rins.c.o CMakeFiles/scip.dir/scip/heur_rootsoldiving.c.o CMakeFiles/scip.dir/scip/heur_rounding.c.o CMakeFiles/scip.dir/scip/heur_shiftandpropagate.c.o CMakeFiles/scip.dir/scip/heur_shifting.c.o CMakeFiles/scip.dir/scip/heur_simplerounding.c.o CMakeFiles/scip.dir/scip/heur_subnlp.c.o CMakeFiles/scip.dir/scip/heur_sync.c.o CMakeFiles/scip.dir/scip/heur_trivial.c.o CMakeFiles/scip.dir/scip/heur_trivialnegation.c.o CMakeFiles/scip.dir/scip/heur_trustregion.c.o CMakeFiles/scip.dir/scip/heur_trysol.c.o CMakeFiles/scip.dir/scip/heur_twoopt.c.o CMakeFiles/scip.dir/scip/heur_undercover.c.o CMakeFiles/scip.dir/scip/heur_vbounds.c.o CMakeFiles/scip.dir/scip/heur_veclendiving.c.o CMakeFiles/scip.dir/scip/heur_zeroobj.c.o CMakeFiles/scip.dir/scip/heur_zirounding.c.o CMakeFiles/scip.dir/scip/message_default.c.o CMakeFiles/scip.dir/scip/nodesel_bfs.c.o CMakeFiles/scip.dir/scip/nodesel_breadthfirst.c.o CMakeFiles/scip.dir/scip/nodesel_dfs.c.o CMakeFiles/scip.dir/scip/nodesel_estimate.c.o CMakeFiles/scip.dir/scip/nodesel_hybridestim.c.o CMakeFiles/scip.dir/scip/nodesel_restartdfs.c.o CMakeFiles/scip.dir/scip/nodesel_uct.c.o CMakeFiles/scip.dir/scip/presol_milp.cpp.o CMakeFiles/scip.dir/scip/presol_boundshift.c.o CMakeFiles/scip.dir/scip/presol_convertinttobin.c.o CMakeFiles/scip.dir/scip/presol_domcol.c.o CMakeFiles/scip.dir/scip/presol_dualagg.c.o CMakeFiles/scip.dir/scip/presol_dualcomp.c.o CMakeFiles/scip.dir/scip/presol_dualinfer.c.o CMakeFiles/scip.dir/scip/presol_gateextraction.c.o CMakeFiles/scip.dir/scip/presol_implics.c.o CMakeFiles/scip.dir/scip/presol_inttobinary.c.o CMakeFiles/scip.dir/scip/presol_qpkktref.c.o CMakeFiles/scip.dir/scip/presol_redvub.c.o CMakeFiles/scip.dir/scip/presol_sparsify.c.o CMakeFiles/scip.dir/scip/presol_dualsparsify.c.o CMakeFiles/scip.dir/scip/presol_stuffing.c.o CMakeFiles/scip.dir/scip/presol_trivial.c.o CMakeFiles/scip.dir/scip/presol_tworowbnd.c.o CMakeFiles/scip.dir/scip/prop_dualfix.c.o CMakeFiles/scip.dir/scip/prop_genvbounds.c.o CMakeFiles/scip.dir/scip/prop_nlobbt.c.o CMakeFiles/scip.dir/scip/prop_obbt.c.o CMakeFiles/scip.dir/scip/prop_probing.c.o CMakeFiles/scip.dir/scip/prop_pseudoobj.c.o CMakeFiles/scip.dir/scip/prop_redcost.c.o CMakeFiles/scip.dir/scip/prop_rootredcost.c.o CMakeFiles/scip.dir/scip/prop_symmetry.c.o CMakeFiles/scip.dir/scip/prop_vbounds.c.o CMakeFiles/scip.dir/scip/prop_sync.c.o CMakeFiles/scip.dir/scip/reader_bnd.c.o CMakeFiles/scip.dir/scip/reader_ccg.c.o CMakeFiles/scip.dir/scip/reader_cip.c.o CMakeFiles/scip.dir/scip/reader_cnf.c.o CMakeFiles/scip.dir/scip/reader_cor.c.o CMakeFiles/scip.dir/scip/reader_dec.c.o CMakeFiles/scip.dir/scip/reader_diff.c.o CMakeFiles/scip.dir/scip/reader_fix.c.o CMakeFiles/scip.dir/scip/reader_fzn.c.o CMakeFiles/scip.dir/scip/reader_gms.c.o CMakeFiles/scip.dir/scip/reader_lp.c.o CMakeFiles/scip.dir/scip/reader_mps.c.o CMakeFiles/scip.dir/scip/reader_mst.c.o CMakeFiles/scip.dir/scip/reader_opb.c.o CMakeFiles/scip.dir/scip/reader_osil.c.o CMakeFiles/scip.dir/scip/reader_pip.c.o CMakeFiles/scip.dir/scip/reader_pbm.c.o CMakeFiles/scip.dir/scip/reader_ppm.c.o CMakeFiles/scip.dir/scip/reader_rlp.c.o CMakeFiles/scip.dir/scip/reader_sol.c.o CMakeFiles/scip.dir/scip/reader_sto.c.o CMakeFiles/scip.dir/scip/reader_smps.c.o CMakeFiles/scip.dir/scip/reader_tim.c.o CMakeFiles/scip.dir/scip/reader_wbo.c.o CMakeFiles/scip.dir/scip/reader_zpl.c.o CMakeFiles/scip.dir/scip/sepa_cgmip.c.o CMakeFiles/scip.dir/scip/sepa_clique.c.o CMakeFiles/scip.dir/scip/sepa_closecuts.c.o CMakeFiles/scip.dir/scip/sepa_aggregation.c.o CMakeFiles/scip.dir/scip/sepa_convexproj.c.o CMakeFiles/scip.dir/scip/sepa_disjunctive.c.o CMakeFiles/scip.dir/scip/sepa_eccuts.c.o CMakeFiles/scip.dir/scip/sepa_gauge.c.o CMakeFiles/scip.dir/scip/sepa_gomory.c.o CMakeFiles/scip.dir/scip/sepa_impliedbounds.c.o CMakeFiles/scip.dir/scip/sepa_intobj.c.o CMakeFiles/scip.dir/scip/sepa_mcf.c.o CMakeFiles/scip.dir/scip/sepa_oddcycle.c.o CMakeFiles/scip.dir/scip/sepa_rapidlearning.c.o CMakeFiles/scip.dir/scip/sepa_strongcg.c.o CMakeFiles/scip.dir/scip/sepa_zerohalf.c.o CMakeFiles/scip.dir/scip/treemodel.c.o CMakeFiles/scip.dir/scip/benders_default.c.o CMakeFiles/scip.dir/scip/benderscut_feas.c.o CMakeFiles/scip.dir/scip/benderscut_feasalt.c.o CMakeFiles/scip.dir/scip/benderscut_int.c.o CMakeFiles/scip.dir/scip/benderscut_nogood.c.o CMakeFiles/scip.dir/scip/benderscut_opt.c.o CMakeFiles/scip.dir/scip/table_default.c.o CMakeFiles/scip.dir/scip/benders.c.o CMakeFiles/scip.dir/scip/benderscut.c.o CMakeFiles/scip.dir/scip/bendersdefcuts.c.o CMakeFiles/scip.dir/scip/boundstore.c.o CMakeFiles/scip.dir/scip/branch.c.o CMakeFiles/scip.dir/scip/clock.c.o CMakeFiles/scip.dir/scip/concsolver.c.o CMakeFiles/scip.dir/scip/concurrent.c.o CMakeFiles/scip.dir/scip/conflict.c.o CMakeFiles/scip.dir/scip/conflictstore.c.o CMakeFiles/scip.dir/scip/cons.c.o CMakeFiles/scip.dir/scip/cutpool.c.o CMakeFiles/scip.dir/scip/cuts.c.o CMakeFiles/scip.dir/scip/debug.c.o CMakeFiles/scip.dir/scip/dialog.c.o CMakeFiles/scip.dir/scip/disp.c.o CMakeFiles/scip.dir/scip/event.c.o CMakeFiles/scip.dir/scip/fileio.c.o CMakeFiles/scip.dir/scip/heur.c.o CMakeFiles/scip.dir/scip/heuristics.c.o CMakeFiles/scip.dir/scip/compr.c.o CMakeFiles/scip.dir/scip/history.c.o CMakeFiles/scip.dir/scip/implics.c.o CMakeFiles/scip.dir/scip/interrupt.c.o CMakeFiles/scip.dir/scip/intervalarith.c.o CMakeFiles/scip.dir/scip/lp.c.o CMakeFiles/scip.dir/scip/matrix.c.o CMakeFiles/scip.dir/scip/mem.c.o CMakeFiles/scip.dir/scip/misc.c.o CMakeFiles/scip.dir/scip/misc_linear.c.o CMakeFiles/scip.dir/scip/misc_nonlinear.c.o CMakeFiles/scip.dir/scip/nlp.c.o CMakeFiles/scip.dir/scip/nodesel.c.o CMakeFiles/scip.dir/scip/paramset.c.o CMakeFiles/scip.dir/scip/presol.c.o CMakeFiles/scip.dir/scip/presolve.c.o CMakeFiles/scip.dir/scip/pricestore.c.o CMakeFiles/scip.dir/scip/pricer.c.o CMakeFiles/scip.dir/scip/primal.c.o CMakeFiles/scip.dir/scip/prob.c.o CMakeFiles/scip.dir/scip/prop.c.o CMakeFiles/scip.dir/scip/reader.c.o CMakeFiles/scip.dir/scip/relax.c.o CMakeFiles/scip.dir/scip/reopt.c.o CMakeFiles/scip.dir/scip/retcode.c.o CMakeFiles/scip.dir/scip/scip_bandit.c.o CMakeFiles/scip.dir/scip/scip_benders.c.o CMakeFiles/scip.dir/scip/scip_branch.c.o CMakeFiles/scip.dir/scip/scip_compr.c.o CMakeFiles/scip.dir/scip/scip_concurrent.c.o CMakeFiles/scip.dir/scip/scip_conflict.c.o CMakeFiles/scip.dir/scip/scip_cons.c.o CMakeFiles/scip.dir/scip/scip_copy.c.o CMakeFiles/scip.dir/scip/scip_cut.c.o CMakeFiles/scip.dir/scip/scip_datastructures.c.o CMakeFiles/scip.dir/scip/scip_debug.c.o CMakeFiles/scip.dir/scip/scip_dcmp.c.o CMakeFiles/scip.dir/scip/scip_dialog.c.o CMakeFiles/scip.dir/scip/scip_disp.c.o CMakeFiles/scip.dir/scip/scip_event.c.o CMakeFiles/scip.dir/scip/scip_expr.c.o CMakeFiles/scip.dir/scip/scip_general.c.o CMakeFiles/scip.dir/scip/scip_heur.c.o CMakeFiles/scip.dir/scip/scip_lp.c.o CMakeFiles/scip.dir/scip/scip_mem.c.o CMakeFiles/scip.dir/scip/scip_message.c.o CMakeFiles/scip.dir/scip/scip_nlp.c.o CMakeFiles/scip.dir/scip/scip_nodesel.c.o CMakeFiles/scip.dir/scip/scip_nonlinear.c.o CMakeFiles/scip.dir/scip/scip_numerics.c.o CMakeFiles/scip.dir/scip/scip_param.c.o CMakeFiles/scip.dir/scip/scip_presol.c.o CMakeFiles/scip.dir/scip/scip_pricer.c.o CMakeFiles/scip.dir/scip/scip_prob.c.o CMakeFiles/scip.dir/scip/scip_probing.c.o CMakeFiles/scip.dir/scip/scip_prop.c.o CMakeFiles/scip.dir/scip/scip_randnumgen.c.o CMakeFiles/scip.dir/scip/scip_reader.c.o CMakeFiles/scip.dir/scip/scip_relax.c.o CMakeFiles/scip.dir/scip/scip_reopt.c.o CMakeFiles/scip.dir/scip/scip_sepa.c.o CMakeFiles/scip.dir/scip/scip_sol.c.o CMakeFiles/scip.dir/scip/scip_solve.c.o CMakeFiles/scip.dir/scip/scip_solvingstats.c.o CMakeFiles/scip.dir/scip/scip_table.c.o CMakeFiles/scip.dir/scip/scip_timing.c.o CMakeFiles/scip.dir/scip/scip_tree.c.o CMakeFiles/scip.dir/scip/scip_validation.c.o CMakeFiles/scip.dir/scip/scip_var.c.o CMakeFiles/scip.dir/scip/scipdefplugins.c.o CMakeFiles/scip.dir/scip/scipcoreplugins.c.o CMakeFiles/scip.dir/scip/scipgithash.c.o CMakeFiles/scip.dir/scip/scipshell.c.o CMakeFiles/scip.dir/scip/sepa.c.o CMakeFiles/scip.dir/scip/sepastore.c.o CMakeFiles/scip.dir/scip/set.c.o CMakeFiles/scip.dir/scip/sol.c.o CMakeFiles/scip.dir/scip/solve.c.o CMakeFiles/scip.dir/scip/stat.c.o CMakeFiles/scip.dir/scip/symmetry.c.o CMakeFiles/scip.dir/scip/syncstore.c.o CMakeFiles/scip.dir/scip/table.c.o CMakeFiles/scip.dir/scip/tree.c.o CMakeFiles/scip.dir/scip/var.c.o CMakeFiles/scip.dir/scip/visual.c.o CMakeFiles/scip.dir/scip/rbtree.c.o CMakeFiles/scip.dir/tclique/tclique_branch.c.o CMakeFiles/scip.dir/tclique/tclique_coloring.c.o CMakeFiles/scip.dir/tclique/tclique_graph.c.o CMakeFiles/scip.dir/dijkstra/dijkstra.c.o CMakeFiles/scip.dir/xml/xmlparse.c.o CMakeFiles/scip.dir/objscip/objbenders.cpp.o CMakeFiles/scip.dir/objscip/objbenderscut.cpp.o CMakeFiles/scip.dir/objscip/objbranchrule.cpp.o CMakeFiles/scip.dir/objscip/objconshdlr.cpp.o CMakeFiles/scip.dir/objscip/objdialog.cpp.o CMakeFiles/scip.dir/objscip/objdisp.cpp.o CMakeFiles/scip.dir/objscip/objeventhdlr.cpp.o CMakeFiles/scip.dir/objscip/objheur.cpp.o CMakeFiles/scip.dir/objscip/objmessagehdlr.cpp.o CMakeFiles/scip.dir/objscip/objnodesel.cpp.o CMakeFiles/scip.dir/objscip/objpresol.cpp.o CMakeFiles/scip.dir/objscip/objpricer.cpp.o CMakeFiles/scip.dir/objscip/objprobdata.cpp.o CMakeFiles/scip.dir/objscip/objprop.cpp.o CMakeFiles/scip.dir/objscip/objreader.cpp.o CMakeFiles/scip.dir/objscip/objrelax.cpp.o CMakeFiles/scip.dir/objscip/objsepa.cpp.o CMakeFiles/scip.dir/objscip/objtable.cpp.o CMakeFiles/scip.dir/objscip/objvardata.cpp.o CMakeFiles/scip.dir/nlpi/nlpi.c.o CMakeFiles/scip.dir/nlpi/nlpi_all.c.o CMakeFiles/scip.dir/nlpi/nlpioracle.c.o CMakeFiles/scip.dir/nlpi/expr.c.o CMakeFiles/scip.dir/nlpi/intervalarithext.cpp.o CMakeFiles/scip.dir/nlpi/nlpi_ipopt_dummy.c.o CMakeFiles/scip.dir/nlpi/nlpi_worhp_dummy.c.o CMakeFiles/scip.dir/nlpi/nlpi_filtersqp_dummy.c.o CMakeFiles/scip.dir/nlpi/exprinterpret_cppad.cpp.o CMakeFiles/scip.dir/lpi/lpi_spx2.cpp.o CMakeFiles/scip.dir/tpi/tpi_none.c.o CMakeFiles/scip.dir/symmetry/compute_symmetry_bliss.cpp.o  -o ../../bin/scip -Wl,-rpath,:::::::::::::: /usr/lib/x86_64-linux-gnu/libz.so /usr/lib/x86_64-linux-gnu/libreadline.so /usr/lib/x86_64-linux-gnu/libgmp.so /usr/lib/x86_64-linux-gnu/libgmpxx.so ../../lib/libsoplex.a /usr/lib/x86_64-linux-gnu/libbliss.so /usr/lib/x86_64-linux-gnu/libgmp.so /usr/lib/x86_64-linux-gnu/libgmpxx.so /usr/lib/x86_64-linux-gnu/libz.so /usr/local/lib/libtbb.a /usr/lib/x86_64-linux-gnu/libgmp.so /usr/lib/x86_64-linux-gnu/libgmpxx.so -lquadmath ../../papilo/libclusol.a /usr/lib/x86_64-linux-gnu/libopenblas.so -lgfortran -lquadmath

/usr/bin/ld: /usr/local/lib/libtbb.a(governor.o): undefined reference to symbol 'pthread_key_delete@@GLIBC_2.2.5'
//lib/x86_64-linux-gnu/libpthread.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status                                                                                                                                                                     scip/src/CMakeFiles/scip.dir/build.make:9182: recipe for target 'bin/scip' failed
make[2]: *** [bin/scip] Error 1
make[2]: Leaving directory '/opt/scipoptsuite-7.0.1/papBuild'
CMakeFiles/Makefile2:1639: recipe for target 'scip/src/CMakeFiles/scip.dir/all' failed 
make[1]: *** [scip/src/CMakeFiles/scip.dir/all] Error 2
make[1]: Leaving directory '/opt/scipoptsuite-7.0.1/papBuild'
Makefile:162: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by alexhoen** [comment]
Hi ssriram1992,
sorry that it took so long to respond and to fix the error. 
We located the error and provided a fix with the new SCIPOptSuite Release 7.0.2. 
Can you please check the update and let us know if this help. 
King regards and sorry for the inconvenience
